webpack.config.js

```diff
@@ -4,15 +4,17 @@ const MonacoWebpackPlugin = require("monaco-editor-webpack-plugin");
 const TerserPLugin = require("terser-webpack-plugin");
 const CopyPlugin = require("copy-webpack-plugin");
 const { ProvidePlugin } = require("webpack");
+const { version } = require("./package.json");
 
 const OUTPUT_FOLDER = path.join(__dirname, "build");
 const CODEBASE = path.join(__dirname, "source");
 
 const { NODE_ENV, RELEASE_VERSION, MODE } = process.env;
 const isProd = NODE_ENV === "prod";
+const wfVersion = RELEASE_VERSION || version;
```

Comment:
outside of the release flow we don't have version number -> need to fall back when env var is empty

Reply:
I think we do, the `RELEASE_VERSION` env var is not specific to the releasing Workflow, it is app-level. We can assume it will always be set, just like we do with the `MODE` env var.

Reply:
problem is it failed without this. For some reason `RELEASE_VERSION` was empty on the CI and on locally as well. Not sure why but I could not get it to work without this. But since we have a way to use package json easily from here I can't see a huge issue or is it @erosdome ?